### PR TITLE
chore(cuda): Remove old comments from concrete-cuda.

### PR DESCRIPTION
**Pull request by pdroalves**
### Description
There were many legacy comments that are not relevant or useful anymore. This PR removes them.

### Checklist 

(Use '[x]' to check the checkboxes, or submit the PR and then click the checkboxes)

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [x] The PR description links to the related issue (to link an issue, use '#XXX'.)
* [ ] The tests on AWS have been launched and are successful (comment with @slab-ci cpu_test and/or @slab-ci gpu_test to trigger the tests)
* [ ] The draft release description has been updated
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by pdroalves** [comment]
@slab-ci gpu_test

**Comment by pdroalves** [comment]
@slab-ci gpu_test

File: concrete-cuda/cuda/src/bootstrap_low_latency.cuh [hunk 1]
```diff
@@ -222,17 +215,6 @@ __global__ void device_bootstrap_low_latency(
...
-      // todo(Joao): **cannot use this optimization**
-      // the reason is that one of the input ciphertexts (blockIdx.z)
-      // might skip an iteration while others don't, which as a result
-      // will make that block not call the grid.sync(), causing a deadlock;
-      // maybe it's a workaround to add grid.sync() here, but not sure if
-      // there are any edge cases?
-
-      // continue
-    }
-
```
Comment (agnesLeroy) [inline concrete-cuda/cuda/src/bootstrap_low_latency.cuh:235]: I'm not sure we should remove this block: @bbarbakadze wdyt?

**Review by agnesLeroy — COMMENTED** [review]
Hello @pdroalves! Thanks a lot for this PR, I just have a doubt about one code block. @bbarbakadze I remember we had kept it on purpose, od you remember whyN

**Review by bbarbakadze — APPROVED** [review]

**Pull request merged**